.travis.yml

```diff
@@ -26,13 +57,17 @@ matrix:
   exclude:
     - env: BOGUS_JOB=true
 
+  allow_failures:
+    - env: TEST_SUITE=compile_fail
```

Comment:
Does this allow bjam builds to fail as well?  If so it shouldn't do so.

Reply:
Not, that should not be there. My bad. Removed.